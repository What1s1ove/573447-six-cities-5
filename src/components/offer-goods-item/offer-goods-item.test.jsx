import * as React from 'react';
import {render} from 'enzyme';
import OfferGoodsItem from './offer-goods-item';

describe(`OfferGoodsItem component`, () => {
  it(`should render OfferGoodsItem component`, () => {
    const component = render(<OfferGoodsItem good="TV" />);

    expect(component).toMatchSnapshot();
  });
});
