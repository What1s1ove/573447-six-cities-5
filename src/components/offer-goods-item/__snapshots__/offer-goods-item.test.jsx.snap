// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OfferGoodsItem component should render OfferGoodsItem component 1`] = `
<li
  class="property__inside-item"
>
  TV
</li>
`;
